### ISSUE TITLE
PIP install fails 

### ISSUE THREAD

**Issue by vabiro** [Tapawingo/FreeTakServer#65]
Hi,

I am attempting to install the server, and when I try to install using pip I receive the following:
```Collecting FreeTAKServer
  Using cached FreeTAKServer-0.8.23.tar.gz (29 kB)
    ERROR: Command errored out with exit status 1:
     command: /usr/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-2IOpnC/FreeTAKServer/setup.py'"'"'; __file__='"'"'/tmp/pip-install-2IOpnC/FreeTAKServer/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-FOeVdj
         cwd: /tmp/pip-install-2IOpnC/FreeTAKServer/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-2IOpnC/FreeTAKServer/setup.py", line 4, in <module>
        with open(path.join(this_directory, 'README.md'), encoding='utf-8') as f:
    TypeError: 'encoding' is an invalid keyword argument for this function
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.```

I have Python 3.4.2 and Python 2.7.9 

Any thoughts on what could be causing this?

Victor

**Comment by brothercorvo** [comment]
read the instructions: you need Python 3.6+

**Comment by naman108** [comment]
try to use pip3

**Comment by korvyr** [comment]
Yup, installation instructions likely should be amended a bit, but you should be using pip3 and python3.

**Comment by brothercorvo** [comment]
solved in 0.8.50
